src/bidiServer/index.ts

```diff
@@ -89,30 +91,64 @@ function parseArguments() {
  */
 async function onNewBidiConnectionOpen(
   headless: boolean,
-  chromeChannel: ChromeReleaseChannel,
+  chromeChannel: 'dev' | 'canary' | 'beta' | 'stable',
```

Comment:
Double-check: Is empty string included here? It doesn't seem like so: http://shortn/_264g710TVY

Reply:
ChromeReleaseChannel | undefined are allow here.